wordpress-mobile/WordPress-Android
# Support media deep links

This PR adds support for media deeplinks as a part of the Super fast media uploads
- `wordpress.com/media/{domain name}`
- `intent://media/#Intent;scheme=jetpack;package=com.jetpack.android`


## To Test:
**Preparing**
- Remove other versions of the app from your device (WP and JP)
- Download the links zip file, unzip and update the test-media-links.html file to include your domain name.
[test-media-links.html.zip](https://github.com/wordpress-mobile/WordPress-Android/files/13640040/test-media-links.html.zip)
- Save the update html file and download to the device you will be using to test this PR
- Download and install JP app from this PR
- Login

**Testing the deep links**
- Open the downloaded `test-media-links.html` in a browser on your device
- Tap on the `Intent Media` link 
- ✅ Verify the app is launched and the media view is visible for the selected site
- Tap on the `wordpress.com/media` link
- ✅ Verify the app is launched and the media view is visible for the selected site
- Tap on the `wordpress.com/media/nonsense` link
- ✅ Verify the app is launched and the media view is visible for the selected site
- Tap on the `wordpress.com/media/..` for your domain link
- ✅ Verify the app is launched and the media view is visible for your domain

-----

## Regression Notes

1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A

-----

## PR Submission Checklist:

-----

## UI Changes Testing Checklist:N/A
